spec/features/participants/participant_list/identity_creates_participant_spec.rb

```diff
@@ -3,47 +3,83 @@
 feature 'User creates Participant', js: true do
```

Comment:
I'm not a fan of this testing style. Capybara already reads like English. 

Also, is a feature spec absolutely necessary here? SPARC-Fulfillment test suite runs painfully slow as it is.